### PR TITLE
Update shadowsocks.md

### PR DESCRIPTION
**Pull request by vxtls**
Add sample configuration to help users better configure the program.

### CODE REVIEW

File: docs/v5/config/proxy/shadowsocks.md [hunk 1]
```diff
@@ -54,3 +54,38 @@ outbound.shadowsocks
...
+        "loglevel": "info"
+    },
+    "inbounds": [
+        {
+            "port": 443,
+            "listen": "0.0.0.0",
+            "protocol": "shadowsocks",
+            "settings": {
+                "method": "aes-256-gcm", //这里使用`aes-256-gcm`加密方式，如果你的CPU不带aes加密指令集也可以使用`"chacha20-poly1305"` 或 `"chacha20-ietf-poly1305"`
+                "password": "ukrxmqlkYTnyyb5sgc8mqiDe5vMoFqRwxefqiigxfdzgigLASE=", //更改这里的密码，让他变得更加复杂
```
Comment (xiaokangwang) [inline docs/v5/config/proxy/shadowsocks.md]: Would you mind enlighten me the reason "他" is chosen among 它他她㸰袉?

**Review by xiaokangwang — COMMENTED** [review]
I think your merge request is in general in good shape, however there is an unrecognized character choice that requires justification.

File: docs/v5/config/proxy/shadowsocks.md (hunk 1 above)
Comment (vxtls) [reply docs/v5/config/proxy/shadowsocks.md]: Sorry, this is a typing error.

**Comment by xiaokangwang** [comment]
Thanks! I have merged it.

**Pull request merged**